Sources/Common/Extensions/DateExtension.swift

```diff
@@ -54,6 +54,10 @@ public extension Date {
         return Calendar.current.isDate(date1, inSameDayAs: date2)
     }
 
+    static func isSameHour(_ date1: Date, _ date2: Date) -> Bool {
+        return Calendar.current.isDate(date1, equalTo: date2, toGranularity: .hour)
```

Comment:
```suggestion
        Calendar.current.isDate(date1, equalTo: date2, toGranularity: .hour)
```

Reply:
I'm actually going to delete it because it's not used anywhere :) thanks for spotting.